### PR TITLE
Fix bug when 'Enter' key is pressed very quickly

### PR DESCRIPTION
**Pull request by Xatpy**
“defensive method”.
The open dialog method was being called twice, so it’swrongly).

Please, @javisantana , could you review it?
When you have a chance, I guess that just right now it's not a good time because of Odyssey integration. :)

### CODE REVIEW

**Comment by viddo** [comment]
retest this please

**Comment by Cartofante** [comment]
The front end tests were OK

**Comment by javisantana** [comment]
@Xatpy better other person review it since I helped with this

**Comment by xavijam** [comment]
I'll do.

File: lib/assets/javascripts/cartodb/models/table.js [hunk 1]
```diff
@@ -313,18 +313,20 @@
...
-            self.trigger('columnDelete', columnName);
-            self.notice(self._TEXTS.columnDeleted, 'info');
-            self.fetch();
-          },
-          error: function(e, resp) {
-            self.error('error deleting column', resp);
-          },
-          wait: true
-      });
+      if (c !== undefined) {
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb/models/table.js]: lacks of test

**Comment by javisantana** [comment]
@Xatpy the changes on models should be tested

**Comment by saleiva** [comment]
bump!

**Comment by Xatpy** [comment]
@javisantana I thought that with this [test](https://github.com/CartoDB/cartodb/blob/master/lib/assets/test/spec/cartodb/models/table.spec.js#L541) was enough.

**Comment by Xatpy** [comment]
@javisantana  Would it be a correct test?
```
    it("should not remove a column if it's invalid", function() {
      var server = sinon.fakeServer.create();
      server.respondWith("DELETE", "/api/v1/tables/" + table.get('name') + "/columns/test",
        [200, { "Content-Type": "application/json" },
       '{"name":"irrelevant1","type":"text","cartodb_type":"string"}']);
      var succeded = false;
      table.bind('columnDelete', function() {
        succeded = true;
      });
      table.deleteColumn('');
      server.respond();
      expect(succeded).not.toBeTruthy();
    });
```

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1317/))

**Pull request merged**